examples/pypi-find-links/pixi.toml

```diff
@@ -3,14 +3,14 @@ name = "pypi-find-links"
 authors = ["Tim de Jager <tim@prefix.dev>"]
 channels = ["conda-forge"]
 platforms = ["osx-arm64", "osx-64", "linux-64", "win-64"]
+[project.pypi-options]
```

Comment:
What do you think about adding an example of pypi options per feature?


and to explain what will happen if we have an environment with two feature that have different pypi options, who will <<win>> ?

Reply:
I'll take this on in a seperate improve docs PR.